third_party/maya/lib/usdMaya/util.cpp

```diff
@@ -1106,7 +1106,10 @@ UsdMayaUtil::MDagPathToUsdPath(
     std::replace(usdPathStr.begin(), usdPathStr.end(), '|', '/');
     std::replace(usdPathStr.begin(), usdPathStr.end(), ':', '_'); // replace namespace ":" with "_"
 
-    SdfPath usdPath(usdPathStr);
+SdfPath
```

Comment:
Hey @elrond79, it looks like there might have been some kind of weird merge error here? It looks like the MDagPathToUsdPath function stomped over some part of the above function.

Reply:
And also, is this new function needed? I didn't see it called in this change anywhere..